tests/edge-cases/apply-02.txt

```diff
@@ -0,0 +1,2 @@
+brun '(a (q 1))'
+FAIL: unimplemented operator 3
```

Comment:
this is a bit questionable as well, but it seems odd to have `a` in the operator list at all, before it's implemented

Reply:
No problem. This documents that it doesn't work in this branch.
I think that's good.